client/src/components/atoms/SearchableDropdown.tsx

```diff
@@ -1,21 +1,25 @@
-import React, { FunctionComponent, useEffect, useState } from "react";
+import React, { FunctionComponent, useState } from "react";
```

Comment:
Modified the behaviour of `SearchableDropdown`. It no longer keeps track of selected string or search string in it's own state. It now derives these values from passed props.

I performed regression checks on the other component that uses `SearchableDropdown` (i.e. `RequestForm`). I was able to select/search request groups and request types using the new `SearchableDropdown`.